src/pipeline/CommonBindings.cpp

```diff
@@ -53,7 +86,54 @@ void CommonBindings::bind(pybind11::module& m){
         .def_readwrite("average", &CpuUsage::average)
         .def_readwrite("msTime", &CpuUsage::msTime)
     ;
+    // CameraModel enum bindings
+    py::enum_<CameraModel>(m, "CameraModel", DOC(dai, CameraModel))
+        .value("Perspective", CameraModel::Perspective)
+        .value("Fisheye", CameraModel::Fisheye)
+        .value("Equirectangular", CameraModel::Equirectangular)
+        .value("RadialDivision", CameraModel::RadialDivision)
+    ;
 
+    // StereoRectification
+    py::class_<StereoRectification> (m, "StereoRectification", DOC(dai, StereoRectification))
+        .def(py::init<>())
+        .def_readwrite("rectifiedRotationLeft", &StereoRectification::rectifiedRotationLeft)
+        .def_readwrite("rectifiedRotationRight", &StereoRectification::rectifiedRotationRight)
+        .def_readwrite("leftCameraSocket", &StereoRectification::leftCameraSocket)
+        .def_readwrite("rightCameraSocket", &StereoRectification::rightCameraSocket)
+        ;
+
+    // Extrinsics 
+    py::class_<Extrinsics> (m, "Extrinsics", DOC(dai, Extrinsics))
+        .def(py::init<>())
+        .def_readwrite("rotationMatrix", &Extrinsics::rotationMatrix)
+        .def_readwrite("translation", &Extrinsics::translation)
+        .def_readwrite("measuredTranslation", &Extrinsics::specTranslation)
```

Comment:
Rename "measuredTranslation"

Reply:
Yikes. Somehow missed these two. Will fix it.